app/models/user.js

```diff
@@ -4,7 +4,7 @@ export default class UserModel extends Model {
   @attr first_name;
   @attr last_name;
   @attr username;
-  @attr('string', { defaultValue: 'active' }) status;
+  @attr('string', { defaultValue: 'onboarding' }) status;
```

Comment:
is this will be in smallcase or capital?

Reply:
Small case as ember docs have also small case
https://guides.emberjs.com/release/models/defining-models/

Reply:
default value is what we want basically so we want smallcase right?

Reply:
yes